### PR TITLE
RouteGuideClient example: use E7 format for latitude & longitude

### PR DESCRIPTION
**Pull request by chalin**
As indicated in the proto file, "Points are represented as latitude-longitude pairs in the E7 representation".

@srini100

### CODE REVIEW

**Comment by srini100** [comment]
The change looks fine to me but I see that other languages like C++ and Python are also using 0 and 1. Not sure what the history is on this. Assigning to @ejona86 to decide.

**Comment by chalin** [comment]
Yeah, probably isn't essential, since this part of the example code is independent of the rest. This issue arose because we print normalized degrees in the Kotlin version of the code, and it seemed a bit strange to see 1e-7 as a degree (FYI).

File: examples/src/main/java/io/grpc/examples/routeguide/RouteGuideClient.java [hunk 1]
```diff
@@ -218,8 +218,8 @@ public void onCompleted() {
 
     try {
       RouteNote[] requests =
-          {newNote("First message", 0, 0), newNote("Second message", 0, 1),
-              newNote("Third message", 1, 0), newNote("Fourth message", 1, 1)};
+          {newNote("First message", 0, 0), newNote("Second message", 0, 10000000),
+              newNote("Third message", 10000000, 0), newNote("Fourth message", 10000000, 10000000)};
```
Comment (dapengzhang0) [inline examples/src/main/java/io/grpc/examples/routeguide/RouteGuideClient.java:222]: nit: it's hard to count how many zeros are there. `10_000_000` might be better.

Comment (chalin) [reply examples/src/main/java/io/grpc/examples/routeguide/RouteGuideClient.java]: Done

**Comment by chalin** [comment]
Test file updated too.

**Comment by chalin** [comment]
All checks are passing now.

**Comment by chalin** [comment]
Anything else to be done before this is merged?

**Comment by dapengzhang0** [comment]
Ping @ejona86 

>The change looks fine to me but I see that other languages like C++ and Python are also using 0 and 1. Not sure what the history is on this. Assigning to @ejona86 to decide.

**Comment by ejona86** [comment]
The other languages should be fixed as well, but I don't want organizing that to hold up this PR longer.

**Pull request merged**